controller/cache/info.go

```diff
@@ -386,3 +398,27 @@ func populateHostNodeInfo(un *unstructured.Unstructured, res *ResourceInfo) {
 		SystemInfo: node.Status.NodeInfo,
 	}
 }
+
+func populateRSInfo(un *unstructured.Unstructured, res *ResourceInfo) {
+	rs := appsv1.ReplicaSet{}
+	err := runtime.DefaultUnstructuredConverter.FromUnstructured(un.Object, &rs)
+	if err != nil {
+		return
+	}
+	var replicas string
+	if rs.Spec.Replicas == nil {
+		replicas = fmt.Sprintf("-/%d/%d", rs.Status.ReadyReplicas, rs.Status.Replicas)
+	} else {
+		replicas = fmt.Sprintf("%d/%d/%d", *rs.Spec.Replicas, rs.Status.ReadyReplicas, rs.Status.Replicas)
```

Comment:
I'm not sure there's a better solution, but IMO it's not good UX to have unlabeled numbers like this. At the very least we should document this somewhere

Reply:
yeah, I'm also not 100% committed on it, will exclude from this change set